python/pyspark/sql/functions.py

```diff
@@ -350,7 +350,7 @@ def corr(col1, col2):
 
     >>> a = range(20)
     >>> b = [2 * x for x in range(20)]
-    >>> df = spark.createDataFrame(zip(a, b), ["a", "b"])
+    >>> df = spark.createDataFrame(__builtin__.zip(a, b), ["a", "b"])
```

Comment:
why did you add `__builtin__` here? This is causing the build failure

Reply:
Before the addition the tests were breaking in the same line, I thought that `zip` being a python built-in function was the cause